src/python/dxpy/scripts/dx.py

```diff
@@ -2514,7 +2514,15 @@ def install(args):
 def uninstall(args):
     app_desc = get_app_from_path(args.app)
     if app_desc is None:
-        parser.exit(1, 'Could not find the app\n')
+        try:
+            dxpy.api.app_uninstall(args.app)
+            print('Uninstalled application with id: ' + args.app)
```

Comment:
From @ptn24:
Please keep the error messages consistent. For example, we refer to apps as "apps" to end users, not as "applications". Furthermore, to preserve the syntax of the existing error message, I think it will be better to use the error messages
`print("Uninstalled the {id} app".format(id=args.app))`
and
`print("Uninstalled the {name} app".format(name=args.app))`
for the two code execution flows.